### ISSUE TITLE
Mapping Vulca V2

### ISSUE THREAD

**Issue by AlexandreRousselet** [Vulca/website#35]
Today, People can see: 
--> Vulca Tour 

For the next version, I would like to help our community to identify the right EU partners. 
For this, main informations are: 
- [ ] What is the experience of the Makerspace based on passed projects
- Local workshop
- Regional or National distributed workshop
- European & More distributed workshop

- [ ] Which Position
- Leader
- Partner
- Affiliated (Expert Advisor) 

- [ ] Which EU Thematic was that project (means: DG) 
- Listing the 53 DG "clickable"

Below a list of example, we can use to make a first new version of the map:
![Speakers-For-Seminar](https://user-images.githubusercontent.com/45399609/68576398-4c39ee00-046e-11ea-9b7a-6d14eb6f8b14.jpg)

Basically today, Vulca knows pretty well the M.Hackerspace Network --> The Field 
If we continue to developp our network in different EU institution we become a stakeholder and such a map can improve our visibility on this. 

Anyway, at the end, everyone needs us for direct connection. 
It is much better to be introduced by vulca guys than sending a random email on the ...contact@....com

**Comment by Thomas0sanz** [comment]
My answer:

If you mean mapping all the network created, and the project-related, It's real research, a real job and a real project.

About the interest of Vulca on that:

1.  If you want to help makers "to identify the right EU partners", map SDG's is a solution because it's from a group of interest that we can connect people not because of pasted project in the one all the team was fill in, 
2. If you want to highlight our knowledge of the network, the map you propose is another thing. Because this map will be to highlight what happened in the past.

It's why, for me, there is a gap between the solution you propose and the issue you highlight.

For me creating a map of the network is very interesting and could be an objective of Vulca. But it's not the main objective of Vulca for the moment, and neither to create connexion.

The map we created was to highlight our community. 

If now the job of Vulca is to map another thing, we have to think differently to Vulca.
